Sources/Extensions/CALayer+Extensions.swift

```diff
@@ -112,14 +117,19 @@ extension CALayer {
         return width
     }
 
-    func updateLayerFrame(for index: Int, size: CGSize, multilineSpacing: CGFloat, paddingInsets: UIEdgeInsets, isRTL: Bool) {
+    func updateLayerFrame(for index: Int, totalLines: Int, size: CGSize, multilineSpacing: CGFloat, paddingInsets: UIEdgeInsets, alignment: NSTextAlignment, isRTL: Bool) {
         let spaceRequiredForEachLine = size.height + multilineSpacing
         let newFrame = CGRect(x: paddingInsets.left,
                               y: CGFloat(index) * spaceRequiredForEachLine + paddingInsets.top,
                               width: size.width,
                               height: size.height)
-        
+
         frame = flipRectForRTLIfNeeded(newFrame, isRTL: isRTL)
+
+        // Align last line according to layer text alignment if not RTL
+        if !isRTL && index == totalLines - 1 && totalLines != 1 {
```

Comment:
Hey @Juanpe 
I'm not sure this is the correct approach. Don't you think that `textAlignment` should be a priority regarding the position of the rectangles? If we define the alignment as `left`, `right`, `center`, it should be so. But if the alignment is `natural`, then it should reflect the current `RTL` state. ([reference](https://developer.apple.com/library/archive/documentation/MacOSX/Conceptual/BPInternational/SupportingRight-To-LeftLanguages/SupportingRight-To-LeftLanguages.html))

I don't know if this would bring any undesired friction to what you already have and the RTL feature itself... So please let me know what you think would be best! 🙇🏼‍♀️

Reply:
Mm...good question 🤔 TBH, I think we should prioritize RTL for `natural` alignments for now. So, we could leave this code there.

Reply:
Yes, we should prioritize RTL for `natural` alignments, but this means that we shouldn't for `left`, `right`, and `center` alignments! My suggestion would be to bundle `flipRectForRTLIfNeeded` and `alignLastLineRect` in a function that would check the alignment and then if `natural`, check RTL. This function would return the final frame.

Having two functions that can do the same looks quite confusing and might cause bugs eventually 🐛 💥

Reply:
Yes, it might make sense. In fact, I'm thinking about how the `flipRect...` method applies to all lines and it'd really make sense to apply it to only the last line as well.

Any suggestion for the name of this method? maybe `transformFrameBasedOnTextAlignment`?

Reply:
Indeed, we could make it more efficient by only applying it to the last line. I was thinking on bundling up RTL in the function so I guess that name wouldn't be 100% accurate... Perhaps simply
`func alignLayerFrame(_ rect: CGRect, alignment: NSTextAlignment, isRTL: Bool) -> CGRect`
...since its parent method is called `updateLayerFrame` 🤷‍♀️

Reply:
LGTM :)